File: algo/problems/0001-two-sum/scala/TwoSumTest.scala
//> using scala 3.4.2
//> using test.dep org.scalameta::munit::1.0.0

class TwoSumTest extends munit.FunSuite:
  test("basic")  { assertEquals(TwoSum.twoSum(Array(2, 7, 11, 15), 9).toList, List(0, 1)) }
  test("middle") { assertEquals(TwoSum.twoSum(Array(3, 2, 4), 6).toList, List(1, 2)) }
  test("same")   { assertEquals(TwoSum.twoSum(Array(3, 3), 6).toList, List(0, 1)) }

File: algo/problems/0001-two-sum/js/twoSum.test.js
const { twoSum } = require('./twoSum');

test('basic', () => { expect(twoSum([2, 7, 11, 15], 9)).toEqual([0, 1]); });
test('middle', () => { expect(twoSum([3, 2, 4], 6)).toEqual([1, 2]); });
test('same value', () => { expect(twoSum([3, 3], 6)).toEqual([0, 1]); });
